### PR TITLE
luakit: 2.2.1 -> 2.3

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/browsers/luakit/default.nix [hunk 1]
```diff
@@ -1,34 +1,56 @@
...
+    gst-plugins-good
+    gst-plugins-bad
+    gst-plugins-ugly
+    gst-libav
+  ]);
+
 
+  # build-utils/docgen/gen.lua:2: module 'lib.lousy.util' not found
+  # TODO: why is not this the default? The test runner adds
+  # ';./lib/?.lua;./lib/?/init.lua' to package.path, but the build-utils
```
Comment (r-burns) [inline pkgs/applications/networking/browsers/luakit/default.nix:51]:
```suggestion
  # ';./lib/?.lua;./lib/?/init.lua' to package.path, but the build-utils
```
fixes editorconfig check

File: pkgs/applications/networking/browsers/luakit/default.nix [hunk 2]
```diff
@@ -52,6 +74,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
+   homepage    = "https://luakit.github.io/";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/luakit/default.nix:77]:
```suggestion
    homepage = "https://luakit.github.io/";
```

File: pkgs/applications/networking/browsers/luakit/default.nix [hunk 3]
```diff
@@ -48,10 +70,11 @@ stdenv.mkDerivation rec {
       --prefix XDG_CONFIG_DIRS : "$out/etc/xdg"
       --prefix LUA_PATH ';' "${luaKitPath};$LUA_PATH"
       --prefix LUA_CPATH ';' "$LUA_CPATH"
-    )
+                                                                          )
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/luakit/default.nix:73]:
```suggestion
    )
```

**Comment by jgarte** [comment]
@AndersonTorres Should [luasocket](https://search.nixos.org/packages?channel=unstable&show=lua53Packages.luasocket&from=0&size=50&sort=relevance&query=luasocket) be including in the build inputs since it is required by the new gopher module in 2.3 or should it be something that the user should install separately?

See here under _Update Information_ in the luakit 2.3 changelog: https://github.com/luakit/luakit/releases/tag/2.3

**Comment by AndersonTorres** [comment]
AFAIR luasocket had some linking problems, and I disabled it.
Have you tried?

**Pull request merged**